### PR TITLE
Database Models

### PR DESCRIPTION
**Pull request by htdvisser**

### CODE REVIEW

File: models/game.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+teamSchema.plugin(timestamps);
+
+// AskedQuestion Schema
+var askedQuestionSchema = new Schema({
+  question: { type: Schema.Types.ObjectId, required: true, ref: 'Question' },
+  team: { type: Number, required: true },
+  askedAt: { type: Date, required: true },
+  deadlineAt: { type: Date, required: true },
+  answeredAt: { type: Date },
+  answer: { type: Number }
```
Comment (janvandekerkhof) [inline models/game.js]: Also add answeredBy?

File: models/game.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+askedQuestionSchema.plugin(timestamps);
+
+// Game Schema
+var gameSchema = new Schema({
+  gameType: { type: String, required: true, enum: ['quiz'] },
+  startedAt: { type: Date },
+  room: { type: Schema.Types.ObjectId, required: true, ref: 'Room' },
+  users: [{ type: Schema.Types.ObjectId, ref: 'User' }],
+  teams: [teamSchema],
+  questions: [askedQuestionSchema],
```
Comment (janvandekerkhof) [inline models/game.js]: Rename to `askedQuestions`? As we also have next `nextQuestions`

File: test/models/question.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    forEach(question.answers, function(val) { val.correct = false; });
+    question.validate(function(err) {
+      should.exist(err);
+      err.should.have.property('errors');
+      err.errors.should.not.have.property('question');
+      err.errors.should.have.property('answers');
+      done();
+    });
+  });
+
```
Comment (janvandekerkhof) [inline test/models/question.js]: Add a test for when there are multiple correct answers, should also validate to false

File: models/game.js (hunk 1 above)
Comment (htdvisser) [inline models/game.js]: Good one.

File: models/game.js (hunk 2 above)
Comment (htdvisser) [inline models/game.js]: I'll rename it to ``previousQuestions``.

File: test/models/question.js (hunk 3 above)
Comment (htdvisser) [inline test/models/question.js]: Added.

**Pull request merged**